src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -628,6 +629,149 @@ JSON example:
+| BackgroundSuppressionNumberPulses | OPTIONAL, RECOMMENDED if `BackgroundSuppression` is `true`                    | [number][]                              | The number of background suppression pulses used. Note that this excludes any effect of background suppression pulses applied before the labeling.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| BackgroundSuppressionPulseTime    | OPTIONAL, RECOMMENDED if `BackgroundSuppression` is `true`                    | [array][] of [numbers][]                | Array of numbers containing timing, in seconds, of the background suppression pulses with respect to the start of the labeling. In case of multi-PLD with different background suppression pulse times, only the pulse time of the first PLD should be defined.                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| VascularCrushingVENC              | OPTIONAL, RECOMMENDED if `VascularCrushing` is `true`                         | [number][] or \[array][] of [numbers][] | The crusher gradient strength, in centimeters per second. Specify either one number for the total time-series, or provide an array of numbers, for example when using QUASAR, using the value zero to identify volumes for which `VascularCrushing` was turned off. Corresponds to DICOM Tag 0018,925A `ASL Crusher Flow Limit`.                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Here you wrote `\[array][]`, but the leading backslash prevents a proper link rendering, see this screenshot:

![grafik](https://user-images.githubusercontent.com/9084751/102772111-44de5700-4387-11eb-9036-3a738795faa0.png)


you just need to remove the backslash, and adjust the table pipes.

Reply:
@sappelhoff This should be fixed now! Thanks for spotting it.